--- hdfeos5/src/EHapi.c	2017-09-15 05:14:51.557540000 -0500
+++ hdfeos5/src/EHapi.c	2017-09-15 05:15:40.487270607 -0500
@@ -11379,7 +11379,7 @@
   return(-1);
 }
 
-
+#endif /* H5_HAVE_FILTER_SZIP */
 
 /*----------------------------------------------------------------------------|
 |  BEGIN_PROLOG                                                               |
@@ -11509,7 +11509,6 @@
     }
 }
 
-#endif /* H5_HAVE_FILTER_SZIP */
 
 
 #ifndef __cplusplus
