--- ispc.cpp    2018-04-03 01:13:58.555407000 +0000
+++ ispc.cpp    2018-04-03 01:17:18.026243382 +0000
@@ -59,7 +59,10 @@
   #include <llvm/IR/Module.h>
   #include <llvm/IR/Instructions.h>
 #endif
-#if ISPC_LLVM_VERSION >= ISPC_LLVM_3_6 // LLVM 3.6+
+#if ISPC_LLVM_VERSION >= ISPC_LLVM_6_0 // LLVM 6.0+
+  #include <llvm/CodeGen/TargetSubtargetInfo.h>
+  #include <llvm/CodeGen/TargetLowering.h>
+#elif ISPC_LLVM_VERSION >= ISPC_LLVM_3_6 // LLVM 3.6+
   #include <llvm/Target/TargetSubtargetInfo.h>
   #if ISPC_LLVM_VERSION >= ISPC_LLVM_3_7 // LLVM 3.7+
     #include <llvm/Target/TargetLowering.h>
