--- arch/arch-XC30_Cray.fcm	2017-10-02 09:46:48.308432000 +0000
+++ arch/arch-XC30_Cray.fcm	2017-10-02 19:25:16.226734177 +0000
@@ -16,27 +16,27 @@
 
 %CCOMPILER      CC
 %FCOMPILER      ftn
-%LINKER         CC
+%LINKER         ftn
 
-%BASE_CFLAGS    -DMPICH_SKIP_MPICXX -h msglevel_4 -h zero -h gnu
+%BASE_CFLAGS
 
 ## Only use -O3 if you can load module cce/8.3.7 or later
-#%PROD_CFLAGS    -O3 -DBOOST_DISABLE_ASSERTS
+%PROD_CFLAGS    -O3 -DBOOST_DISABLE_ASSERTS
 
 ## Otherwise take your pick of these, refer to information above.
-%PROD_CFLAGS    -O1 -DBOOST_DISABLE_ASSERTS
+## %PROD_CFLAGS    -O1 -DBOOST_DISABLE_ASSERTS
 ## %PROD_CFLAGS    -Gfast -DBOOST_DISABLE_ASSERTS
 %DEV_CFLAGS     -O2
 %DEBUG_CFLAGS   -g 
 
-%BASE_FFLAGS    -em -m 4 -e0 -eZ
+%BASE_FFLAGS    
 %PROD_FFLAGS    -O3
 %DEV_FFLAGS     -G2
 %DEBUG_FFLAGS   -g 
 
 %BASE_INC       -D__NONE__
-%BASE_LD        -D__NONE__
+%BASE_LD        -D__NONE__ -lnetcdff_parallel -cxxlib
 
-%CPP            cpp
-%FPP            cpp -P -CC
+%CPP            CC -E
+%FPP            ftn -E
 %MAKE           gmake
